src/WebJobs.Script.Grpc/Abstractions/WorkerDescriptionExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions
+{
+    public static class WorkerDescriptionExtensions
+    {
+        public static bool IsValid(this WorkerDescription workerDescription)
+        {
+            if (string.IsNullOrEmpty(workerDescription.Language))
+            {
+                throw new ArgumentNullException(nameof(workerDescription.Language));
```

Comment:
`ArgumentNullException` isn't appropriate here. Data annotations has a `ValidationException` we could use here

Reply:
Once this is moved to the new configuration model, there's built in support for options validation (along with an appropriate exception type as well)